index.html

```diff
@@ -93,80 +98,172 @@ <h2>Atomic Operations</h2>
       <section id="atomic-operations-introduction" class="informative">
         <h2>Introduction</h2>
         <p>
-          Fedora provides factilities by which to conduct multiple operations against a repository
-          in an atomic fashion. This is accomplished by the use of headers on requests. An atomic
-          series of operations should not require any more requests than the same series of
-          operations performed non-atomically.
+          Fedora provides factilities by which to conduct multiple operations
+          against a repository in an atomic fashion.
+        </p>
+        <pre class="http">
+          # Discover the transaction endpoint
+
+          HEAD http://example.org/foo
+
+          200 OK
+          Link: &lt;http://example.org/foo/fcr:tx>; rel="[URI TBD]"
+
+          # Begin a transaction
+
+          POST http://example.org/foo/fcr:tx
+
+          201 Created
+          Location: http://example.org/foo/fcr:tx/01234567-89ab-cdef-0123-456789abcdef
+
+          # Create a new resource within the transaction
+
+          POST http://example.org/foo
+          Atomic-ID: http://example.org/foo/fcr:tx/01234567-89ab-cdef-0123-456789abcdef
+
+          201 Created
+          Location: http://example.org/foo/abcdef01-2345-6789-abcd-ef0123456789
+          Link: &lt;http://example.org/foo/fcr:tx/01234567-89ab-cdef-0123-456789abcdef>; rel="[URI TBD]"
```

Comment:
I think I like these changes over the previous version, where transaction handling isn't a secondary effect of a regular request (which could get weird. why make a GET just to abort a transaction?). 

Providing a URI for the transaction for interaction purposes seems good, and so it makes sense to pass the tx uri in. It also implies that a transaction is bound to a particular domain, in case we ever end up with fedora clustering. That's probably realistic. 

I guess a Link header makes sense, although I don't think it'd be unreasonable to use a custom header either.

Reply:
Yeah, the "GET to abort a transaction" always bothered me, too.

As for the `Link` header, the rel should mean something like "this representation is this resource as viewed in the transaction identified by the link". I retained the custom `Atomic-ID` header for the requests since I didn't see anything in the standard HTTP set of headers that seemed appropriate for the meaning of "do this in the context of".